text/2019-11-07-raft-witness.md

```diff
@@ -0,0 +1,85 @@
+# Raft Witness
+
+## Summary
+
+This RFC proposes a new role Witness to Raft, and describes how it could benefit us.
```

Comment:
This section should describe a summary of the feature you want to add.

For example:

> Add the Witness role to Raft. Witness roles only store log metadata, not the entries themselves, and allow a cluster to greatly reduce traffic and storage. While this feature can reduce resiliency, it many cases it is desirable to reduce cluster load and energy consumption.